lib/JSON/Validator/Error.pm

```diff
@@ -34,7 +34,8 @@ our $MESSAGES = {
     additionalProperties => 'Properties not allowed: %3.',
     maxProperties        => 'Too many properties: %3/%4.',
     minProperties        => 'Not enough properties: %3/%4.',
-    required             => 'Missing property.',
+    required             => 'Missing required property.',
+    dependencies         => 'Missing dependent property.',
```

Comment:
I think this is incomplete: It should mention the path/name of the dependent property. How about something like "%3 missing dependent property." or something? If you can make it shorter or rephrase it so "%3" comes at the end, then that would be super.

Reply:
See https://github.com/mojolicious/json-validator/pull/197#discussion_r379681124